underfs/gcs/src/main/java/alluxio/underfs/gcs/GCSUnderFileSystem.java

```diff
@@ -347,13 +347,13 @@ public void setMode(String path, short mode) throws IOException {}
   // Not supported
```

Comment:
I think the comment is still accurate. We do not propagate any user/group/mode info from gcs. This PR changes the default to an empty value which is accepted as undefined from a null value which would be invalid.